test/sqlite/result_helper.cpp

```diff
@@ -309,7 +309,10 @@ bool TestResultHelper::CheckStatementResult(const Statement &statement, ExecuteC
 			error = !error;
 		}
 		if (result.HasError() && !statement.expected_error.empty()) {
-			if (!StringUtil::Contains(result.GetError(), statement.expected_error)) {
+			// We run both comparions on purpose, we might move to only the second but might require some changes in
```

Comment:
If I grok, the change is to check both raw error match and kw-subst'd match, and the comment suggests that we may not need both... but unclear why -- I assume it's uncertainty/lack of confidence that all cases are covered? Might be nice clarify for future code/comment GC
(ditto for comment below @ line 313)